registry/handlers/app.go

```diff
@@ -43,7 +43,7 @@ const randomSecretSize = 32
 type App struct {
 	context.Context
 
-	Config configuration.Configuration
+	Config *configuration.Configuration
```

Comment:
`Config` is supposed to be an embed of the configuration. It allows the app to copy in the configuration and not change the callers version. Any specific reason to do this?